CMakeLists.txt

```diff
@@ -961,7 +962,20 @@ elseif (MICROSOFT)
 else ()
 	find_package(Threads REQUIRED)
 	set(PTHREAD_LIBRARY ${CMAKE_THREAD_LIBS_INIT})
-endif()
+    message(STATUS "Pthread library: ${PTHREAD_LIBRARY}")
+endif()
+
+# To avoid the need for other judgments when ENABLE_STDCXX_SYNC is OFF in the future, this is a separate conditional statement.
+if (NOT ENABLE_STDCXX_SYNC AND ENABLE_MONOTONIC_CLOCK)
+    list(PREPEND CMAKE_REQUIRED_LIBRARIES "${PTHREAD_LIBRARY}")
+    check_symbol_exists(pthread_condattr_setclock "pthread.h" HAVE_PTHREAD_CONDATTR_SETCLOCK)
+    if (HAVE_PTHREAD_CONDATTR_SETCLOCK)
+        add_definitions(-DHAVE_PTHREAD_CONDATTR_SETCLOCK=1)
+    else ()
+        message(WARNING, "Can't find pthread_condattr_setclock. Try -DENABLE_STDCXX_SYNC=ON or -DENABLE_MONOTONIC_CLOCK=OFF; the latter is not recommended though.")
+    endif ()
+endif ()
+
```

Comment:
Indent using tabs, please.

Reply:
Maybe `add_definitions(-DHAVE_PTHREAD_CONDATTR_SETCLOCK=0)` otherwise? Not necessary though. Just a thought.

Reply:
Not necessary. If this function doesn't exist, monotonic clock should be simply off, as it's not usable, and SRT with this enforced won't be simply usable at all.